services/web/server/src/simcore_service_webserver/users_api.py

```diff
@@ -103,6 +103,11 @@ async def get_user_profile(app: web.Application, user_id: UserID) -> ProfileGet:
         "organizations": user_standard_groups,
         "all": all_group,
     }
+
+    # TODO: conversion from db fields to ProfileGet is error-prone. Add ProfileGet.from_db_model() ??
```

Comment:
sonarcloud won't like this

Reply:
nope sorry!... i missed that one ... you were faster reviewing!